modules/cnf-topology-aware-lifecycle-operator-operator-update.adoc

```diff
@@ -204,18 +213,22 @@ $ oc get jobs,pods -n openshift-talo-pre-cache
 +
 [source,terminal]
 ----
-$ oc get cgu cgu-upgrade-complete -ojson | jq '.status.conditions'
+$ oc get cgu -n default cgu-upgrade-complete -ojsonpath='{.status.conditions}'
 ----
 +
 .Example output
```

Comment:
The example output does not match the context. This is an out for upgrade completion, while we are expecting an output that indicates precaching completed (but upgrade has not started).

Reply:
This looks good to me.

Reply:
Updated